compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -207,11 +209,17 @@ object Completion {
      * considered.
      */
     def addMemberCompletions(qual: Tree)(implicit ctx: Context): Unit = {
-      addAccessibleMembers(qual.tpe)
-      if (!mode.is(Mode.Import)) {
-        // Implicit conversions do not kick in when importing
-        implicitConversionTargets(qual)(ctx.fresh.setExploreTyperState())
-          .foreach(addAccessibleMembers)
+      if (qual.tpe =:= defn.NothingType)
+        ()
+      else if (qual.tpe =:= defn.NullType)
+        addAccessibleMembers(defn.AnyRefType)
+      else {
+        addAccessibleMembers(qual.tpe)
+        if (!mode.is(Mode.Import)) {
+          // Implicit conversions do not kick in when importing
+          implicitConversionTargets(qual)(ctx.fresh.setExploreTyperState())
+            .foreach(addAccessibleMembers)
+        }
```

Comment:
This makes sense, except is seems a bit hacky?

Reply:
Why hacky? Do you have another idea of how to handle bottom types correctly?

Reply:
Your change suggests `addAccessibleMembers` is buggy on `NullType` and `Nothing`, so why doesn't the fix should go closer to the source of the bug? At least in `accessibleMembers`, one would think.
And while `defn.NothingType` probably needs to be a special case, `defn.NullType` inherits from `AnyRefType`, so should inherit its members, no?

Reply:
Yes, I still don't understand what most of this patch is doing.

Reply:
I refactored the code. Now I only special case implicit completions on `Null`. The description of the PR was updated. The source of the issue where the implicit conversion, not the `addAccessibleMembers`